### ISSUE TITLE
Formatting for "Help" page -- using <h3> for questions isn't readable, can't scan

### ISSUE THREAD

**Issue by jwirfs-brock** [denverfoundation/storybase#364]
I started putting  up some of our FAQ/Help information at http://staging.floodlightproject.org/en/help/

Currently, I'm using `<h2>` (and corresponding side-nav) for the main sections (Building Stories, Reading Stories, etc.) and `<h3>` for the individual questions. Formatted this way, it's really hard to scan. Adding `<br />` between the questions and answers didn't improve the readability, because the `<h2>` and `<h3>`s are still the same color as the body text.

@ghing @patternleaf Perhaps we can fix this with styling -- maybe change the `<h2>` and `<h3>` tags to orange? And if we do that, do we want to do it site-wide, or just on the Help page? 

@bpawlak and @tekhneco how is this done on the Data Engine site? I recall from the demo that the answers were toggle-able but the questions were always visible. We don't need to add anything fancy for launch (it just needs to be readable), but in the future it might be nice to have them be consistent.

**Comment by ghing** [comment]
@jwirfs-brock We can definitely fix this with styles.  

@patternleaf I'd suggest adding `.faq-section` and `.faq-question` classes.  We could make the color of those orange for now and change it later.

**Comment by bpawlak** [comment]
DataEngine simply uses a toggle option on the question.  So by default only the questions are shown and clicking on it toggles the answer open.  Clicking again toggles it closed:  http://198.101.241.26/help

**Comment by jwirfs-brock** [comment]
Just out of curiousity, how hard would the toggle be to implement? We
shouldn't really be adding features now, so I think the solution @ghing
suggested should be good. But post-launch the toggle might be nice
(especially if we want to include screen shots or anything that takes up a
lot of space in the answers).

**Comment by bpawlak** [comment]
it's actually pretty easy to do. We'd have to add a class to the <p> tags for the content (or wrap them in a div) but with jQuery, there's a toggle() method

**Comment by jwirfs-brock** [comment]
Great. I'll defer to @ghing and @patternleaf for whether they want to add
that to the pre-launch plate.

**Comment by ghing** [comment]
I think for launch we should just make sure we mark up the FAQ correctly to make it ready for a jQuery snippet.  Here's the relevant snippet from the Data Engine:

```
(function ($) {
          $(document).ready(function(){
            $('.accordion').find('.faq-title').addClass('ss-navigateright');
            $('.accordion').find('.ds-header').addClass('is-collapsed').click(
              function() {
                $(this).toggleClass('is-collapsed is-expanded').next().toggle('fast');
                $(this).find('.faq-title').toggleClass('ss-navigateright ss-navigatedown');
                return false;
              }
            ).next().hide();
          });
        })(jQuery);
```

@jwirfs-brock, the most important thing to get ready for this is to make sure the answer to the question is wrapped in a `div` element.

When you've finished marking it up, assign this back to me and I'll decide if there's time to add the jQuery.

I'd suggest marking it up like this:

```
<h3 class="faq-question">I want to write a story. Where do I begin?</h3>
<div class="faq-answer">
  <p>Click on “BUILD A STORY” in the Floodlight navigation menu at the top of any page to see a short description of the story building process. From that page, click on the “Create Your Story” button to launch the Story Builder</p>
</div>
```

The most important part is the `faq-question` class on the `h3` and the `div` wrapping the answer.  The `faq-answer` class on the `div` is optional, but adding it gives us a little more styling precision.

**Comment by bpawlak** [comment]
The quick and dirty: 

in CSS
.faq-answer {display:none;}

in javascript
$(".faq-question").click(function() {
    $(this).next(".faq-answer").toggle();
});

**Comment by jwirfs-brock** [comment]
And like this for the section headings?

`<h2 class="faq-section">Building Stories</h2>`

**Comment by jwirfs-brock** [comment]
Ok, all of the current FAQ text has been marked up.

I'll be adding more sections, and when I do I'll make sure they are marked up in the same way.

**Comment by bpawlak** [comment]
You can, yes, if you think we'd be collapsing the sections.  But if we're primarily using the left-nav to jump to sections, we'd want to give it a `<a name="value">` tag.  But since that's been deprecated in HTML5, we now simply use ID  (so it'd be `<h2 class="faq-section" id="building_stories">Building Stories</h2>`
In any case, I think it's a good idea to add the ID attribute to the individual sections so that we can dynamically target it at some point if need be.

**Comment by ghing** [comment]
Cool.  Do we have an icon for doing the expanding/collapsing?

**Comment by jwirfs-brock** [comment]
Not yet. The Data Engine page uses an orange arrow/chevron thing, similar
to what we have on the sub-nav on the interior pages.

**Comment by bpawlak** [comment]
I've always used expand/collapse sprites like (http://documentation.devexpress.com/AspNet/HelpResource.ashx?help=AspNet&document=img10788.jpg)

**Comment by ghing** [comment]
Ok.  The heading colors are now orange on staging.  If I have time to make sprites to match the orange arrow/chevron thing I'll implement the expanding/collapsing.  Otherwise, it's at least more readable now.

**Comment by bpawlak** [comment]
I just sent you a graphic via email

**Comment by ghing** [comment]
@bpawlak, thanks for sending the graphic.  I might go this route though: http://fortawesome.github.com/Font-Awesome/ which is a freely licensed equivalent of the icon font used in the Data Engine site.

**Comment by jwirfs-brock** [comment]
@ghing The Font Awesome solution sounds good to me. Let's do it.

**Comment by ghing** [comment]
Closed by 057089b1a18435b9e4ef87d9a8c329074ca6a107
